runtime/stdlib/builtin.go

```diff
@@ -181,136 +179,10 @@ var CreatePublicKeyFunction = NewStandardLibraryFunction(
-
-The order of the public keys in the slice does not matter since the aggregation is commutative. 
-No subgroup membership check is performed on the input keys.
-The function returns nil if the array is empty or any of the input keys is not a BLS key.
-`
-
-var AggregateBLSPublicKeysFunction = NewStandardLibraryFunction(
-	"AggregateBLSPublicKeys",
-	&sema.FunctionType{
-		Parameters: []*sema.Parameter{
-			{
-				Label:          sema.ArgumentLabelNotRequired,
-				Identifier:     "keys",
-				TypeAnnotation: sema.NewTypeAnnotation(&sema.VariableSizedType{Type: sema.PublicKeyType}),
-			},
-		},
-		ReturnTypeAnnotation: sema.NewTypeAnnotation(&sema.OptionalType{Type: sema.PublicKeyType}),
-	},
-	aggregateBLSPublicKeysFunctionDocString,
-	func(invocation interpreter.Invocation) interpreter.Value {
-		publicKeys := invocation.Arguments[0].(*interpreter.ArrayValue)
-		return AggregateBLSPublicKeys(
-			invocation.Interpreter,
-			invocation.GetLocationRange,
-			publicKeys,
-		)
-	},
-)
-
-func AggregateBLSPublicKeys(
-	inter *interpreter.Interpreter,
-	getLocationRange func() interpreter.LocationRange,
-	publicKeys *interpreter.ArrayValue,
-) interpreter.Value {
-	publicKeyArray := make([]interpreter.MemberAccessibleValue, 0, publicKeys.Count())
-	publicKeys.Iterate(func(element interpreter.Value) (resume bool) {
-		publicKey := element.(interpreter.MemberAccessibleValue)
-		publicKeyArray = append(publicKeyArray, publicKey)
-		return true
-	})
-
-	aggregatedKey, err := inter.AggregateBLSPublicKeysHandler(
-		inter,
-		getLocationRange,
-		publicKeyArray,
-	)
-	if err != nil {
-		panic(err)
-	}
-
-	return aggregatedKey
-}
-
-func AggregateBLSSignatures(
-	inter *interpreter.Interpreter,
-	signatures *interpreter.ArrayValue,
-) interpreter.Value {
-	bytesArray := make([][]byte, 0, signatures.Count())
-	signatures.Iterate(func(element interpreter.Value) (resume bool) {
-		sig := element.(*interpreter.ArrayValue)
-		bytes := make([]byte, 0, sig.Count())
-		sig.Iterate(func(element interpreter.Value) (resume bool) {
-			i := element.(interpreter.UInt8Value)
-			bytes = append(bytes, byte(i))
-			return true
-		})
-		bytesArray = append(bytesArray, bytes)
-		return true
-	})
-
-	aggregatedBytes, err := inter.AggregateBLSSignaturesHandler(
-		bytesArray,
-	)
-
-	// if the crypto layer produces an error, we have invalid input, return nil
-	if err != nil {
-		return interpreter.NilValue{}
-	}
-
-	aggregatedSignature := make([]interpreter.Value, 0, len(aggregatedBytes))
-	for _, b := range aggregatedBytes {
-		aggregatedSignature = append(aggregatedSignature, interpreter.UInt8Value(b))
-	}
-
-	return interpreter.NewSomeValueNonCopying(interpreter.NewArrayValue(
-		inter,
-		interpreter.ByteArrayStaticType,
-		signatures.GetOwner(),
-		aggregatedSignature...,
-	))
-}
-
 // BuiltinValues
 
-func BuiltinValues() StandardLibraryValues {
-	signatureAlgorithmValue := StandardLibraryValue{
+var BuiltinValues = StandardLibraryValues{
```

Comment:
We made this a function sometimes back, to avoid built-in values being shared across interpreters and avoid any potential concurrency issues (https://github.com/onflow/cadence/pull/1096).

Reply:
ah ok, should be fine then.